charts/subgraph-radio/README.md

```diff
@@ -0,0 +1,96 @@
+
+## Introduction
+
+The [Subgraph-Radio](https://docs.graphops.xyz/graphcast/radios/subgraph-radio) a Graphcast Radio focused on sending gossips about particular subgraphs on a P2P network.
+
+## Chart Features
+
+- Actively maintained by [GraphOps](https://graphops.xyz) [and contributors](https://github.com/graphops/launchpad-charts/graphs/contributors)
+- Strong security defaults (non-root execution, ready-only root filesystem, drops all capabilities)
+- Readiness checks to ensure traffic only hits `Pod`s that are healthy and ready to serve requests
+- Support for `ServiceMonitor`s to configure Prometheus to scrape metrics ([prometheus-operator](https://github.com/prometheus-operator/prometheus-operator))
+- Support for configuring Grafana dashboards ([grafana](https://github.com/grafana/helm-charts/tree/main/charts/grafana))
+
+## Enabling inbound P2P dials
+
+By default, your subgraph-radio release will not have an internet-accessible port for P2P traffic. This makes it harder for your node to establish a strong set of peers because you cannot accept inbound P2P dials. To change this behaviour, you can set `p2pNodePort.enabled` to `true`. This will make your node accessible via the Internet using a `Service` of type `NodePort`. When using `p2pNodePort.enabled`, the exposed IP address on your arbitrum ENR record will be the "External IP" of the Node where the Pod is running. When using this mode, `replicaCount` will be locked to `1`.
+
+```yaml
+# values.yaml
+
+p2pNodePort:
+  enabled: true
+  port: 31000 # Must be globally unique and available on the host
+```
+
+## Quickstart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add graphops http://graphops.github.io/launchpad-charts
+$ helm install my-release graphops/subgraph-radio
+```
+
+## Upgrading
+
+We recommend that you pin the version of the Chart that you deploy. You can use the `--version` flag with `helm install` and `helm upgrade` to specify a chart version constraint.
+
+This project uses [Semantic Versioning](https://semver.org/). Changes to the version of the application (the `appVersion`) that the Chart deploys will generally result in a patch version bump for the Chart. Breaking changes to the Chart or its `values.yaml` interface will be reflected with a major version bump.
+
+We do not recommend that you upgrade the application by overriding `image.tag`. Instead, use the version of the Chart that is built for your desired `appVersion`.
+
+## Values
+
+| Key | Description | Type | Default |
+|-----|-------------|------|---------|
+ | affinity |  | object | `{}` |
+ | aliases | Set custom aliases for preconfigured commands in your environment | object | `{}` |
+ | env.GRAPHCAST_NETWORK | Comma separated static list of content topics to subscribe to | string | `"mainnet"` |
```

Comment:
The description here should be updated to something like "The Graphcast Messaging fleet and pubsub namespace to use.", currently it is describing `env.TOPICS`

Do you remember where you got this from in the first place? wonder if we misaligned it somewhere

Reply:
@hopeyen I was getting these values from [here](https://github.com/graphops/subgraph-radio/blob/b6b86353ab8593cd2d3c09b74d87d35048d60e2d/one-shot/src/config.rs#L73C1-L74C65)  - nothing is misaligned it was error on my side